### PR TITLE
Enhancement: Make buttons scrollable

### PR DESCRIPTION
**Pull request by joenunezb**
Adding a simple scrollbar to buttons created via `/buttons` command.  Tested with a simple QR CoT script. I also clicked around with LenAnderson's Codex and did not see any particular regressions. 

No errors reported in console log, tested a few variations of sizes with Chrome's device toolbar and everything looked fine. 

# Before:

PC:

https://github.com/user-attachments/assets/b1ae6403-942f-4ffe-bbb0-6ec66460317b

Mobile (bonus):

https://github.com/user-attachments/assets/0dea11c6-6ac6-4f64-b2bb-1cba5c649efc

# After:

PC:

https://github.com/user-attachments/assets/d8899a87-5081-4cf1-bb03-daecef917fd0

Mobile (bonus):

https://github.com/user-attachments/assets/6db5bd98-73d0-4d31-b1f1-83d32ff7a014

## Checklist:

- [x] I have read the [Contributing guidelines](https://github.com/SillyTavern/SillyTavern/blob/release/CONTRIBUTING.md).

### CODE REVIEW

File: public/scripts/slash-commands.js [hunk 1]
```diff
@@ -2074,7 +2074,34 @@ async function buttonsCallback(args, text) {
             let popup;
 
             const buttonContainer = document.createElement('div');
-            buttonContainer.classList.add('flex-container', 'flexFlowColumn', 'wide100p', 'm-t-1');
+            buttonContainer.classList.add('flex-container', 'flexFlowColumn', 'wide100p');
+
+            const scrollableContainer = document.createElement('div');
+            scrollableContainer.style.maxHeight = '50vh'; // Use viewport height instead of fixed pixels
```
Comment (Cohee1207) [inline public/scripts/slash-commands.js]: Try not to mix styles with code. Move that to a CSS file and set a class.

Comment (joenunezb) [reply public/scripts/slash-commands.js]: Thanks for the review! I'll get to this on the weekend and will push out a commit.

Comment (joenunezb) [reply public/scripts/slash-commands.js]: @Cohee1207 , I've addressed the comment and re-tested. Would you like me to re-upload videos of the feature still working? I tested on mobile and PC.

**Review by Cohee1207 — APPROVED** [review]
Looks good now, thanks.

**Pull request merged**